### PR TITLE
Add Python to $PATH

### PR DESCRIPTION
**Pull request by igorboskovic3**
# Description
New tool, Bug fixing, or Improvement?  
Please include a summary of the change and which issue is fixed. Also include relevant motivation and context.  
**For new tools, please provide total size and installation time.**

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

This is fix for https://github.com/actions/runner-images/issues/6507

### CODE REVIEW

File: images/macos/provision/configuration/environment/bashrc [hunk 1]
```diff
@@ -29,3 +29,5 @@ export HOMEBREW_CASK_OPTS="--no-quarantine"
 
 export BOOTSTRAP_HASKELL_NONINTERACTIVE=1
 export BOOTSTRAP_HASKELL_INSTALL_NO_STACK_HOOK=1
+
+export PATH=/Library/Frameworks/Python.framework/Versions/Current/bin:$PATH
```
Comment (al-cheb) [inline images/macos/provision/configuration/environment/bashrc:33]: nitpick: move this line to the PATH section between 16..20 lines.

**Review by khaledalam173 — CHANGES_REQUESTED** [review]
New

**Review by al-cheb — APPROVED** [review]

**Review by khaledalam173 — COMMENTED** [review]
#

**Pull request merged**